src/op_handlers/near_call.rs

```diff
@@ -42,6 +42,7 @@ pub fn _near_call(vm: &mut VMState, opcode: &Opcode) {
         new_storage,
         callee_ergs,
         new_transient_storage,
+        exception_handler as u64,
```

Comment:
If the spec specifies which bytes from the given U256 are used for the exception handler, let's explicitly cast it like that, else let's leave it as is for now.

Reply:
The exception handler is not received in a register (U256) but from an immediate value (u16), so the conversion to u64 will never fail, it is only done since it needs to be used as the PC later